openslides_backend/action/actions/user/send_invitation_email.py

```diff
@@ -191,10 +191,8 @@ def __missing__(self, key: str) -> str:
             "password": user.get("default_password", ""),
             **subject_format,
         }
-        if meeting_id:
-            body_dict["url"] = mail_data.get("users_pdf_url", "")
-        else:
-            body_dict["url"] = mail_data.get("url", "")
+
+        body_dict["url"] = mail_data.get("url", "")
```

Comment:
move into definition of the `body_dict` above

Reply:
Done.